README.md

```diff
@@ -28,6 +28,15 @@ Ideally this project should be managed with [Docker](https://www.docker.com/get-
 
 Check the [composer.json](composer.json) for required PHP version and other dependencies.
 
+#### Drupal 10.1
```

Comment:
remove this as oe_content is a dev dependency

Reply:
Removed